### PR TITLE
Decentr native token

### PR DESCRIPTION
**Pull request by megaherz**

### CODE REVIEW

**Comment by megaherz** [comment]
```
osmosisd query bank balances osmo12c2mwfzvn2cy462zarr7yg8yj9l8n76mmeckea
balances:
- amount: "1019998"
  denom: ibc/9BCB27203424535B6230D594553F1659C77EC173E36D9CF4759E7186EE747E84
- amount: "35935090"
  denom: uosmo
pagination:
  next_key: null
  total: "0"
```

IBC received transaction https://www.mintscan.io/osmosis/txs/3546CB671464C82AA1D2A2F746D927344CCDE81784DC17066C39473B02D0C74B

File: osmosis-1/osmosis-1.assetlist.json [hunk 1]
```diff
@@ -1067,6 +1067,36 @@
...
+      "denom_units": [
+        {
+          "denom": "ibc/9BCB27203424535B6230D594553F1659C77EC173E36D9CF4759E7186EE747E84",
+          "exponent": 0,
+          "aliases": [
+            "udec"
+          ]
+        },
+        {
+          "denom": "udec",
```
Comment (IbrarMakaveli) [inline osmosis-1/osmosis-1.assetlist.json:1082]: "denom": "dec",

File: osmosis-1/osmosis-1.assetlist.json [hunk 2]
```diff
@@ -1067,6 +1067,36 @@
...
+      "display": "dec",
+      "symbol": "DEC",
+      "ibc": {
+        "source_channel": "channel-1",
+        "dst_channel": "channel-181",
+        "source_denom": "udec"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/decentr.png",
+        "svg": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/decentr.svg"
```
Comment (IbrarMakaveli) [inline osmosis-1/osmosis-1.assetlist.json:1097]: "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/dec.png",
        "svg": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/dec.svg"

**Review by IbrarMakaveli — CHANGES_REQUESTED** [review]

File: osmosis-1/osmosis-1.assetlist.json (hunk 1 above)
Comment (megaherz) [reply osmosis-1/osmosis-1.assetlist.json]: Thx, fixed

File: osmosis-1/osmosis-1.assetlist.json (hunk 2 above)
Comment (megaherz) [reply osmosis-1/osmosis-1.assetlist.json]: Renamed

**Comment by megaherz** [comment]
Sorry, incorrectly rebased, created a new one https://github.com/osmosis-labs/assetlists/pull/103

**Pull request closed without merging**